### PR TITLE
hanko-js switch to new sdk

### PR DESCRIPTION
**Pull request by bjoern-m**

### CODE REVIEW

**Review by aspeteRakete — APPROVED** [review]

File: hanko-js/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ENV PATH /app/node_modules/.bin:$PATH
 COPY package.json ./
 COPY package-lock.json ./
 
-RUN npm ci --silent
```
Comment (aspeteRakete) [inline hanko-js/Dockerfile:9]: Why did you remove the --silent? For debugging reasons or has this some advantage?

Comment (bjoern-m) [reply hanko-js/Dockerfile:9]: There was something wrong with the package-lock file and the output was not very helpful with --silent. But I can add the parameter again if you want me to.

Comment (bjoern-m) [reply hanko-js/Dockerfile]: https://github.com/teamhanko/hanko/pull/189/commits/4506ddb6bcf239512d3d86c021a2d80e87a01b62
